src/about.html

```diff
@@ -0,0 +1,169 @@
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet" />
+
+    <!-- Scripts -->
+    <script type="module" src="./js/index.js"></script>
+  </head>
+  <body>
+    <header class="header">
+      <div class="toolbar">
+        <ul class="toolbar__list container">
+          <li class="toolbar__list-item">
+            <a><img src="./assets/icons/facebook.svg" alt="Facebook Icon" /></a>
+          </li>
+          <li class="toolbar__list-item">
+            <a><img src="./assets/icons/twitter.svg" alt="Twitter Icon" /></a>
+          </li>
+          <li class="toolbar__list-item"><a>English</a></li>
+          <li class="toolbar__list-item"><a>My Page</a></li>
+          <li class="toolbar__list-item"><a>Logout</a></li>
+        </ul>
+      </div>
+      <div class="navbar container">
+        <div class="navbar__logo-wrapper">
+          <a href="./index.html" class="navbar__logo">
+            <img src="./assets/images/logo.png" alt="Logo">
+          </a>
+        </div>
+        <nav>
+          <button class="navbar__menu-button" type="button">
+            <img src="./assets/icons/menu.svg" alt="Menu icon" />
+          </button>
+          <div class="navbar__links-list-container">
+            <button class="navbar__close-button" type="button">
+              <img src="./assets/icons/chevron-left.svg" alt="Close icon" />
+            </button>
+            <ul class="navbar__links-list">
+              <li class="navbar__links-list-item">
+                <a href="./about.html">About</a>
+              </li>
+              <li class="navbar__links-list-item">
+                <a href="./about.html">Contact</a>
+              </li>
+            </ul>
+          </div>
+        </nav>
+        <button class="navbar__cta-button" type="button">
+          Get Your Ticket
+        </button>
+      </div>
+    </header>
+    <section class="headline">
+      <div class="container">
+        <div class="headline__overtitle">"Hello! JavaScript Lovers"</div>
+        <h1 class="headline__title">
+          Global JavaScript Conference <br />
+          2022
+        </h1>
+        <p class="headline__description">
+          JavaScript is #1 on demand programming language. Meet the ones who
+          shape out the now and the future of the prospering nation.
+        </p>
+        <div class="headline__contact">
+          Please contact us per Email for any further questions about CC Global
+          Summit 2015! <br />
+          <a
+            class="headline__contact-link"
+            href="mailto:ccsummit2015@cckorea.org"
+            >ccsummit2015@cckorea.org</a
+          >
+        </div>
+      </div>
+    </section>
+    <section class="section">
```

Comment:
- [ ] Kindly make sure to add the section missing as suggested in the loom video attached.
<img width="440" alt="Screenshot 2022-05-17 at 8 58 04 PM" src="https://user-images.githubusercontent.com/70269507/168879530-28bbab91-b97b-4132-a6c7-c46441da0c9a.png">